sdk/react/panels/videoView.js

```diff
@@ -121,7 +121,13 @@ class VideoView extends React.Component {
   };
 
   onSeekPressed = (isForward) => {
-    const seekValue = (isForward) ? this.props.config.skipControls.skipForwardTime : this.props.config.skipControls.skipBackwardTime * (-1);
+    let configSeekValue = (isForward) ? this.props.config.skipControls.skipForwardTime : this.props.config.skipControls.skipBackwardTime;
+    if (configSeekValue > 99) {
+      configSeekValue = 99;
+    } else if (configSeekValue < 1) {
+      configSeekValue = 1;
+    }
```

Comment:
Can we use smth like this:
Number.prototype.clamp = function(min, max) {
  return Math.min(Math.max(this, min), max);
};
instead of if-else statement?